apps/sms/views/conversation/test/unit/recipients_test.js

```diff
@@ -143,6 +143,24 @@ suite('Recipients', function() {
       assertDeepEqual(recipient, fixture);
     });
 
```

Comment:
> I was following the format of the tests above and below it, 

We may want to have trailing space in the ```suite``` name, so that when test results a printed - you'll have space between suite and test names. Since the test is the "most bottom node" you don't need trailing space for it.

>  Also, where should I put the leading parenthesis?

I just meant that your test name should say explicitly that it handles the case for leading parenthesis only. So instead of naming test as ```recipients.add() parenthesis``` you'd better say ```recipients.add() with leading parenthesis```